content-src/components/MediaPreview/MediaPreview.scss

```diff
@@ -0,0 +1,39 @@
+.video-preview {
+  position: relative;
+  background: $black no-repeat center center;
+  width: 367px;
+  height: 206px;
+  border-radius: 6px;
+  transition: 0.2s opacity;
+  overflow: hidden;
+  margin-top: 20px;
+}
+
+.video-preview::after {
+  content: '';
+  width: 65px;
+  height: 40px;
+  background: url('img/playButton@2x.png') no-repeat center center;
+  background-size: 65px 40px;
+  display: block;
+  left: 50%;
+  top: 50%;
+  position: absolute;
+  margin-left: -32px;
+  margin-top: -20px;
```

Comment:
This centering strategy definitely works, but I would recommend using flexbox centering on `.video-preview` instead since it doesn't involve hard-coding margin values:
```
display: flex;
align-items: center;
justify-content: center;
```